application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -44,7 +44,6 @@ analytics.mostViewedPages.header.date=Date
 analytics.mostViewedPages.header.emptyvalue=-
 analytics.mostViewedPages.header.entryNbActions.hint=The number of actions performed by the users.
 analytics.mostViewedPages.header.entryNbActions=No. of actions
-analytics.mostViewedPages.header.exitRate.hint=The percentage of visits that left the website after viewing this page.
```

Comment:
Where these translations deleted by mistake?

Reply:
Yes, the translations were deleted by mistake